tensorflow_serving/tools/docker/Dockerfile.mkl

```diff
@@ -87,6 +87,12 @@ ENV MKLDNN_VERBOSE=0
 # For more information vist below reference
 # https://www.tensorflow.org/guide/performance/overview#tuning_mkl_for_the_best_performance
 
+# NOTE: As TENSORFLOW_INTRA_OP_PARALLELISM and TENSORFLOW_INTER_OP_PARALLELISM are
+# configured via SessionOptions in tensorflow, these values
+# will override the values configured via TF_NUM_INTEROP_THREADS and TF_NUM_INTRAOP_THREADS
+# environment variables in tensorflow.
+# https://github.com/tensorflow/tensorflow/blob/d1823e2e966e96ee4ea7baa202ad9f292ac7427b/tensorflow/core/common_runtime/process_util.h#L39-L45
```

Comment:
Thanks @karthikvadla Do you think adding the link to the PR that introduced the above change instead of `git` sha1 would be a better idea:
```
https://github.com/tensorflow/tensorflow/commit/d1823e2e966e96ee4ea7baa202ad9f292ac7427b#diff-7e28f90dfe6efa4df7e2b8b8559db698R33
```

Reply:
good idea, thanks @ashahba . Will update that.